pkg/kubelet/kubeletconfig/configsync.go

```diff
@@ -198,11 +197,11 @@ func restartForNewConfig(eventClient v1core.EventsGetter, nodeName string, sourc
 	// we directly log and send the event, instead of using the event recorder,
 	// because the event recorder won't flush its queue before we exit (we'd lose the event)
 	event := makeEvent(nodeName, apiv1.EventTypeNormal, KubeletConfigChangedEventReason, message)
-	klog.V(3).Infof("Event(%#v): type: '%v' reason: '%v' %v", event.InvolvedObject, event.Type, event.Reason, event.Message)
+	klog.V(3).InfoS("Event created", "event", klog.KObj(event), "eventInvolvedObject", event.InvolvedObject, "eventType", event.Type, "eventReason", event.Reason, "eventMessage", event.Message)
 	if _, err := eventClient.Events(apiv1.NamespaceDefault).Create(context.TODO(), event, metav1.CreateOptions{}); err != nil {
-		utillog.Errorf("failed to send event, error: %v", err)
+		klog.ErrorS(err, "Kubelet config controller failed to send event")
 	}
-	utillog.Infof(message)
+	klog.InfoS("Kubelet config controller", "eventMessage", message)
```

Comment:
```suggestion
	klog.InfoS("Kubelet config controller", "message", message)
```

Reply:
@serathius Elana suggested this change. so which one? :D